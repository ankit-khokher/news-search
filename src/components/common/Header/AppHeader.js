import React from "react";

const AppHeader = () => {
  return (
    <div className="App-header">
      <img
        className="App-logo"
        src="https://img01.bt.co.uk/s/assets/191120/images/logo/logo-2018.svg
        "
        alt="logo"
        height="87"
        width="100"
      />
    </div>
  );
};

export default AppHeader;
