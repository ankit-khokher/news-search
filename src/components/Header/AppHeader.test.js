import { shallow } from "enzyme";
import AppHeader from "./AppHeader";

describe("AppHeader", () => {
  const wrapper = shallow(<AppHeader />);

  it("should render correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });

  it("should have image", () => {
    const img = wrapper.find("img");
    expect(img).toHaveLength(1);
  });
});
