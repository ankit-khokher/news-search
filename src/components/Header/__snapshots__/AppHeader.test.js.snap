// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppHeader should render correctly 1`] = `ShallowWrapper {}`;
